src/watchFixPlugin/index.js

```diff
@@ -0,0 +1,37 @@
+const chalk = require('chalk');
+const configOverrides = require('../utils/configOverrides');
+
+class ESLintWatchFixPlugin {
+  constructor({ stdout, config }) {
+    this._stdout = stdout;
+    this._key = config.key || 'F';
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  async run() {
+    const fix = configOverrides.getFix();
+    configOverrides.setFix(!fix);
+    return true;
+  }
+
+  getUsageInfo() {
+    return {
+      key: this._key,
+      prompt: this._getPrompt(),
+    };
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  _getPrompt() {
```

Comment:
why can’t this be a closed over function instead of a fake private method?

Reply:
👍